app/models/AgentTrustsResponse.scala

```diff
@@ -43,8 +44,8 @@ object AgentTrustsResponse {
         response.status match {
           case OK =>
             response.json.as[AgentTrusts] match {
-              case AgentTrusts(Seq(), _) => NotClaimed
-              case agentTrusts => agentTrusts
+                case AgentTrusts(Seq(), _) => NotClaimed
```

Comment:
@jackheslop96 Can we rename the HTTP response object and the instances? It's not specific to only Agents.